pkg/host/machine_info.go

```diff
@@ -38,8 +38,16 @@ func CollectModulesInfo() ([]KernelModule, error) {
 	return machineModulesInfo()
 }
 
+func CollectDirsInfo() (map[string][]string, error) {
```

Comment:
The set of paths (globs) needs to be passed to this function. Otherwise how does it know what paths to scan for each operating system.